make/Makefile.base

```diff
@@ -288,6 +294,16 @@ ECHO = echo
 #
 EXE_SUFFIX =
 
+# in case something goes wrong and we don't have a compiler, try
```

Comment:
Should the default base compiler be clang instead of gcc?

Reply:
I'm not expecting this to ever get run - we could make it a fatal error instead if we wanted. I'm not sure there really is a good option but I'd bet linux64 is slightly more typical of a platform for using Chapel than Mac, but who knows.